icons/android/add-search-engine-24-dark.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path
+        android:fillColor="#F9F9FA"
+        android:fillAlpha=".8"
```

Comment:
We could simplify this to `android:fillColor="#ccf9f9fa`, applies to other icons as well